### DISCUSSION TITLE
Python support?

### DISCUSSION THREAD

**Discussion by lllama** [volarjs/volar.js#28] [General]
Would it be possible to use Volar where Python is the embedded language? I’m looking at something that follows the svelte style of file: a script tag containing Python, an html/jinja template and a style tag with css. Would a language server for such a thing be possible with Volar as the base? Or would need the proxy/remote style instead of the language services style?

**Comment by johnsoncodehk** [comment]
I think a process needs to be started to run python language server or something like that and forward requests from Volar language server via rpc/pipe etc. The amount of work may or may not be substantial, depending on how much reusable code the community has.

Can you share more of your use case? How many users do you think will need it?
